scripts/install_server_from_src.sh

```diff
@@ -15,67 +14,133 @@ set -e
 : ${DOWNLOAD_CLIENT_TEMPLATES:=true}
 : ${PYTHON_INSTALL:=true}
 : ${DOCKER:=false}
+: ${INSTALL_BIN:="install"}
+: ${INSTALL_OPTS:="-p -m644"}
+: ${SRC_DIR:="/grr"}
+
+function header()
+{
+  echo ""
+  echo "##########################################################################################"
+  echo "     ${*}";
+  echo "##########################################################################################"
+}
+
+OPTIND=1
+while getopts "h?ls:" opt; do
```

Comment:
This should be fixed now.  I wasn't actually using those as environment vars anywhere.  It just kind of evolved weird and didn't get cleaned up.